geode-docs/managing/security/method_invocation_authorizers.html.md.erb

```diff
@@ -0,0 +1,165 @@
+ 
+The methods `getClass`, `readObject`, `readResolve`, `readObjectNoData`, `writeObject` and `writeReplace` are permanently forbidden.
+
+The below table shows those methods that belong to <%=vars.product_name%> and are considered safe (for methods on `org.apache.geode.cache.Region`, the authorizer also verifies that the user has the `DATA:READ:RegionName` permission).
+
+| Class                                     | Allowed Methods                                                                 |
+|-------------------------------------------|---------------------------------------------------------------------------------|
+| `org.apache.geode.cache.Region.Entry`     | `getKey`, `getValue`                                                            |
+| `org.apache.geode.cache.Region`           | `get`, `entrySet`, `keySet`, `values`, `getEntries`, `getValues`, `containsKey` |
+
+### <a id="unrestrictedMethodAuthorizer"></a>UnrestrictedMethodAuthorizer
+
+A flexible `MethodInvocationAuthorizer` that allows any method invocation during the query execution as long as the following conditions are met:
+
+- The method is not considered permanently forbidden by the [RestrictedMethodAuthorizer](method_invocation_authorizers.html#restrictedMethodAuthorizer).
+- The method does not belong to <%=vars.product_name%>, or does belong but it's considered safe by the [RestrictedMethodAuthorizer](method_invocation_authorizers.html#restrictedMethodAuthorizer).
+
+This authorizer implementation addresses only three of the four main security risks: `Java Reflection`, `Cache Modification` and `Region Modification`.
+The `Region Entry Modification` security risk still exists: users with the `DATA:READ:RegionName` permission will be able to execute ANY method (even those that mutate the object) 
+on the entries stored within the region and on instances used as bind parameters of the query, so this authorizer implementation must be used with extreme care.
+
+**Note:**
+Usage of this authorizer is recommended for secured clusters on which only trusted users and applications have access to the query engine.
+It might also be used on clusters on which all entries stored are immutable.
+
+### <a id="javaBeanAccessorMethodAuthorizer"></a>JavaBeanAccessorMethodAuthorizer
+
+A more flexible `MethodInvocationAuthorizer` that allows methods to be invoked during the query execution only if the the following conditions are met:
+
+- The method is not considered permanently forbidden by the [RestrictedMethodAuthorizer](method_invocation_authorizers.html#restrictedMethodAuthorizer).
+- The method does not belong to <%=vars.product_name%>, or does belong but it's considered safe by the [RestrictedMethodAuthorizer](method_invocation_authorizers.html#restrictedMethodAuthorizer).
```

Comment:
Should be "does belong but is considered safe"

Reply:
👍